.changeset/stupid-pumpkins-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
+---
+
+Tester publish
```

Comment:
Do you still want to keep this?

Reply:
Yeah, still want to see if the ternary works